example/life_game/README.md

```diff
@@ -0,0 +1,37 @@
+# Conway's Game of Life
+
+This example is a [Conway's Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life) implementation.
+3 outputs are proposed (console, [OCaml Graphics](https://github.com/ocaml/graphics) and HTML/JavaScript),
+each one using the same WebAssembly modules.
+
+## Console et OCaml Graphics
+
+Console and Graphics implementations use *external functions* technique as the
+[standard example](../). Here are the command lines to run in this directory to execute the programs:
+
+* Console  mode: `dune exec -- ./life_graphics.exe`
+* Graphics mode: `dune exec -- ./life_console.exe`
+
+Several modules (defined in `life.wast` and `life_loop.wast` files) are linked.\
+Why 2 .wast files are required ?\
```

Comment:
> Why are two `.wast` files required ?

Is a more idiomatic formulation

Reply:
Also, maybe put that question in **bold** ? To make it stand out more while you're at it, it's a mere suggestion, do of it what you wish :)